modules/dtg/manifests/backup.pp

```diff
@@ -76,20 +76,22 @@
 # The name of the backup will be used as the directory name for the subdir containing the backups
 # user is the user to ssh in as
 # host is the host to ssh into
-define dtg::backup::hostsetup($user, $host, $weekday) {
+define dtg::backup::hostsetup($user, $group = $dtg::backup::host::user, $host, $weekday) { #needs to have a configurable backup group + default
```

Comment:
I had assumed that this would work given the name of the parameter is passed with the parameter, and can be specified in an arbitrary order. 

There isn't much I can see referring to this in the docs. https://docs.puppetlabs.com/puppet/latest/reference/lang_classes.html